### PR TITLE
9 merchant item disable/enable

### PR DESCRIPTION
**Pull request by amspears007**
Created tests for enable and disable
Form_with button with hidden field for status and item.id
Update method has added conditional to reroute to index page and updated item params with status

### CODE REVIEW

File: app/controllers/merchants/items_controller.rb [hunk 1]
```diff
@@ -16,8 +16,12 @@ def edit
   def update
     show
     if @item.update(item_params)
-      flash[:alert] = "#{@item.name} successfully updated"
-      redirect_to merchant_item_path(@merchant, @item)
+      if params[:commit] == "Submit"
```
Comment (kaileykaes) [inline app/controllers/merchants/items_controller.rb:19]: Clever, Amy.

File: spec/features/merchant/items/index_spec.rb [hunk 2]
```diff
@@ -30,4 +30,38 @@
...
+        save_and_open_page
+        expect(current_path).to eq(merchant_items_path(@betty))
+        # expect(page).to have_content("Status: Enabled")
+        # expect(page).to have_button("Disable")
+      end
+    end
+  end
+end
+
+# US9 Merchant Item Disable/Enable
```
Comment (kaileykaes) [inline spec/features/merchant/items/index_spec.rb]: I'm going to remove the user story comment.

File: spec/models/item_spec.rb [hunk 3]
```diff
@@ -17,6 +17,9 @@
     it {should have_many(:invoice_items)}
     it {should have_many(:invoices).through(:invoice_items)}
 
+    it { should define_enum_for(:status).with_values("Disabled": 0, Enabled: 1)}
+
```
Comment (kaileykaes) [inline spec/models/item_spec.rb:21]: I think one less space here.

**Review by kaileykaes — APPROVED** [review]

File: app/views/merchants/items/index.html.erb [hunk 4]
```diff
@@ -1,4 +1,42 @@
...
+
+  <% @items.each do |item| %>
+<section id="enabled-items-<%= item.id%>">
+    <% if item.status == "Enabled" %>
+      <p>
+        Name: <%= link_to  "#{item.name}", "items/#{item.id}" %><br>
+        Status: <%= item.status %>
+
+        <%= form_with url: "/merchants/#{@merchant.id}/items", method: :patch, local: true do |f| %>
+          <%= hidden_field_tag :status, "Disabled" %>
```
Comment (brad2412) [inline app/views/merchants/items/index.html.erb:16]: Nice job!

**Pull request merged**